types/mongodb/test/cursor.ts

```diff
@@ -16,7 +16,7 @@ async function run() {
   cursor = cursor.hint({ age: 1 });
   cursor = cursor.hint('age_1');
   cursor = cursor.limit(1);
-  cursor = cursor.map(result => {});
+  cursor = cursor.map(result => ({ foo: result.foo }));
```

Comment:
You could potentially change this to `bar: result.foo` and check for `bar` later, to see that mapping works ☺️ 

```suggestion
  cursor = cursor.map(result => ({ bar: result.foo }));
```